src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -1042,4 +1045,73 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 	private isExtensionAllowedToBeRecommended(id: string): boolean {
 		return this._allIgnoredRecommendations.indexOf(id.toLowerCase()) === -1;
 	}
-}
+
+	// {{SQL CARBON EDIT}}
+	promptVisualizerExtensions(): void {
```

Comment:
why can't we use the existing recommendation prompt?  is this going to have all the logic around "Don't show again for this type of extension.." that we have in the other notification?